ui/pages/swaps/build-quote/build-quote.js

```diff
@@ -627,44 +627,41 @@ export default function BuildQuote({
                   alt={t('swapSwapSwitch')}
                 />
               </Box>
-              <Typography variant={TypographyVariant.H7} marginTop={0}>
+              <Text variant={TextVariant.bodySm} as="h6" marginTop={0}>
                 {t('stxDescription')}
-              </Typography>
-              <Typography
-                as="ul"
-                variant={TypographyVariant.H7}
-                fontWeight={FONT_WEIGHT.BOLD}
-                marginTop={3}
-              >
-                <li>{t('stxBenefit1')}</li>
-                <li>{t('stxBenefit2')}</li>
-                <li>{t('stxBenefit3')}</li>
-                <li>
-                  {t('stxBenefit4')}
-                  <Typography
-                    as="span"
-                    fontWeight={FONT_WEIGHT.NORMAL}
-                    variant={TypographyVariant.H7}
-                  >
-                    {' *'}
-                  </Typography>
-                </li>
-              </Typography>
-              <Typography
-                variant={TypographyVariant.H8}
+              </Text>
+              <Text as="ul" variant={TextVariant.bodySm} marginTop={3}>
+                <strong>
+                  <li>{t('stxBenefit1')}</li>
+                  <li>{t('stxBenefit2')}</li>
+                  <li>{t('stxBenefit3')}</li>
+                  <li>
+                    {t('stxBenefit4')}
+                    <Text
+                      as="span"
+                      fontWeight={FontWeight.Normal}
+                      variant={TextVariant.bodySm}
+                    >
+                      {' *'}
+                    </Text>
+                  </li>
+                </strong>
+              </Text>
+              <Text
+                variant={TextVariant.bodyXs}
+                as="h6"
                 color={TextColor.textAlternative}
                 boxProps={{ marginTop: 3 }}
               >
                 {t('stxSubDescription')}&nbsp;
-                <Typography
+                <Text
                   as="span"
-                  fontWeight={FONT_WEIGHT.BOLD}
-                  variant={TypographyVariant.H8}
+                  variant={TextVariant.bodyXs}
                   color={TextColor.textAlternative}
                 >
```

Comment:
Here we could get rid of this tag entirely because all it was doing before was bolding the text. So:
```js
              <Text
                variant={TextVariant.bodyXs}
                as="h6"
                color={TextColor.textAlternative}
                boxProps={{ marginTop: 3 }}
              >
                {t('stxSubDescription')}&nbsp;<strong>{t('stxYouCanOptOut')}&nbsp;</strong>
             </Text>
```

is functionally equivalent but less verbose

Reply:
Hey @brad-decker ,

Please explain this a bit.
Shall I replace the `<Text>` tag with the above code snippet.

Reply:
Hi there, @brad-decker ,

I get it, you are doing it like 
```
<Text>
  Hello, my name is Inigo Montoya. You killed my father. <strong>Prepare to die!</strong>
</Text>
```

The actual role of `<strong>` tag comes into play here.